docs/src/content/docs/reference/fasta-format.md

```diff
@@ -0,0 +1,30 @@
+---
+title: FASTA format
+---
+
+[_FASTA_](https://en.wikipedia.org/wiki/FASTA_format) is a file format to store sequence data.
+The FASTA format does not have an official definition.
```

Comment:
IMO this isn't useful and is if anything confusing - there are a lot of aspects of FASTA that are fully agreed.